stdlib/random.pyi

```diff
@@ -102,7 +102,7 @@ class SystemRandom(Random):
 
 # ----- random function stubs -----
 
```

Comment:
It's not relevant to this PR, but this fatuous comment really grinds my gears; I'd love it if we can get rid of it while we're here 😆

```suggestion
```